app/main.py

```diff
@@ -16,4 +16,8 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"mutable": [my_favourite_films, marks,
+                                collection_of_coins],
+                    "immutable": [lucky_number, pi, one_is_a_prime_number,
+                                  name, profile_info]
+                    }
```

Comment:
```suggestion
sorted_variables = {
    "mutable": [
        my_favourite_films, marks,
        collection_of_coins
     ],
     ....
```
Fix code-style to make code more readable